### DISCUSSION TITLE
Ground overlays

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#1029] [Ideas]
Is there an example in the project code how to add images which "stick" to the map so that they resize and rotate when the map is zoomed and rotated? I know that there are markers but these do not scale with the zoom.

**Comment by devemux86** [comment]
Do you mean [ground overlays](https://developers.google.com/maps/documentation/javascript/examples/groundoverlay-simple)?

**Reply by demiantres** [reply]
Yes.
